service/codegen/src/main/java/io/helidon/service/codegen/ServiceDescriptorCodegen.java

```diff
@@ -1062,6 +1063,17 @@ private void singletonInstanceField(ClassModel.Builder classModel, DescribedServ
                 .defaultValueContent("new " + service.descriptorType().className() + "<>()"));
     }
 
+    private void singletonLoggerField(ClassModel.Builder classModel, DescribedService service) {
+        // singleton instance of the descriptor
+        classModel.addField(instance -> instance
+                .accessModifier(AccessModifier.PRIVATE)
+                .isStatic(true)
+                .isFinal(true)
+                .type(TypeName.create("java.lang.System.Logger"))
```

Comment:
Please use `System.Logger.class`, as that is part of JDK and there is no need to abstract it to a TypeName. TypeName would require a new constant + test to verify it is valid, so this is an easier path.